public/api-spec-v1.json

```diff
@@ -0,0 +1,6554 @@
+{
```

Comment:
Not sure we need/want the json spec as well. The composer command and github actions only generate and check the yaml api spec. So theis file can become outdated without us knowing

Reply:
We use the json spec for generating apiSlices in the admin. I will add it to github actions validation steps

Reply:
Have already added it to composer command: https://github.com/os2display/display-api-service/pull/66/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R132